docs/FormFillerButton.md

```diff
@@ -33,6 +33,23 @@ export const ContactEdit = () => (
 );
 ```
 
+You must define a `dataProvider.getCompletion()` method to fetch the completion suggestions from your API. This method must return a promise that resolves to a `{ data: completionString }` object.
+
+For instance, to use the OpenAI Completion API:
```

Comment:
I wouldn't put this snippet here. Instead? I'd add a link to an anchor to the Data Provider requirements section that you've already added at the end

Reply:
That's what we did for `<SmartRichTextInput>` so I thought we wanted the same

Reply:
Do you to remove this section in all ra-ai documentation pages?

Reply:
Let's do it for this component here, then we'll see (in another pr) for the other ra-ai components.

Reply:
@djhi You removed the snippet alright, but you forgot to add a link to the anchor to the Data Provider requirements section as suggested by @fzaninotto

Reply:
Done